src/theme.js

```diff
@@ -1,14 +1,28 @@
+export const breakpoints = {
+  small: "480px",
+  large: "992px",
+  mobileMin: 320,
+  mobileMax: 767
+};
+
+export const radius = {
+  small: "5px",
+  navigationInput: "33px",
+  navigationLinks: "24px"
+};
+
+export const shadow = {
```

Comment:
jeżeli chodzi o strukture theme to jeżeli rozbiłeś na poszczególne właściwości to uważam, że nie ma potrzeby ich eksportu, eksportujemy sam theme, a właściwości jedynie możemy zdefiniować. Nie wiem czy jestem fanką tworzenia tylu zmiennych, raczej bardziej przejrzysta forma to obiekt z właściwościami.

Reply:
Jasne, rozumiem. Osobiście uważam, tą formę za bardziej przejrzystą, ale fatycznie w tym projekcie wystarczy tylko jeden obiekt.